### DISCUSSION TITLE
unknown error

### DISCUSSION THREAD

**Discussion by IndrajeethY** [react-native-community/discussions-and-proposals#537] [Ideas]
PS D:\react-native-location-tracking> npm start 

> LocationTracking@0.0.1 start
> node node_modules/react-native/local-cli/cli.js start

error Invalid regular expression: /(.*\\__fixtures__\\.*|node_modules[\\\]react[\\\]dist[\\\].*|website\\node_modules\\.*|heapCapture\\bundle\.js|.*\\__tests__\\.*)$/: Unterminated character class. Run CLI with --verbose flag for more details.

y am i getting this errorr? wt i did is npx create-react-app location-tracker and then did npm start

**Comment by cortinico** [comment]
> y am i getting this errorr? wt i did is npx create-react-app location-tracker and then did npm start

This is not the right repo to report your issues. Please open them against https://github.com/facebook/create-react-app/issues
